symphony/content/content.blueprintsdatasources.php

```diff
@@ -168,7 +168,7 @@ public function __form()
 
                             // Handle pre Symphony 2.2.1 Static DS's
                         } else {
-                            $fields['static_xml'] = trim($existing->grab());
+                            $fields['static_xml'] = trim($existing->execute());
```

Comment:
Don't be sorry! I had the same thing when I started fixing Scrutinizer and PHPStorm problems, sometimes it doesn't get it quite right :)